src/styles/themes.less

```
@@ -42,6 +42,17 @@
   }
 }
 
+.minty {
+    // text color in buttons are white, but expected to be black
+    .btn, .btn:hover {
+        color: #495057;
+    }
+    // text color in primary buttons gets overridden by the lines above
+    .btn-primary, .btn-primary:hover {
+        color: #fff;
+    }
```

Something like this should work:
_(I don't think the `:hover` part is needed for this change from testing, but I may have missed something)_
```suggestion
    // Button without color class text is light, but expected to be dark
    .btn:not([class*="btn-"]) {
        color: #495057;
    }
```

---

Your solution solves the problem you pointed out. And the hover-part is not needed, with your solution.
I hope i have accepted it (still new to git)